packages/generator/src/renderer.ts

```diff
@@ -139,6 +163,32 @@ function createHelpers(config: any): string {
   });
 }
 
+/**
+ * Render the API Clients markdown file using the Handlebars template
+ * @param {Map<string, WebApiBaseUnit[]>} apiFamilyMap
+ * @param {Object.<string, RestApi[]} apiFamilyConfig
+ * @returns {string} The rendered template
+ */
+export function createApiClients(
+  apiFamilyMap: Map<string, WebApiBaseUnit[]>,
+  apiFamilyConfig: { [key: string]: RestApi[] }
+): string {
+  const apis = Array.from(apiFamilyMap).map(
+    ([family, apiModels]): ApiClientsInfoT => {
+      // Merge model and config into array of objects
+      return apiModels.map((apiModel: WebApiBaseUnitWithEncodesModel, idx) => {
+        return {
+          family, // Included for ease of access within template
+          model: apiModel.encodes as model.domain.WebApi,
+          config: apiFamilyConfig[family][idx]
```

Comment:
Will the index match to the same API always in both arrays? Resolving to apiModel is a async call, just make sure the order is retained

Reply:
The models are generated using `_.map` on the config object, so order is preserved.

https://github.com/SalesforceCommerceCloud/commerce-sdk/blob/c3ec85c59ccdb2172d344cb2b6ea2020e510b6c4/packages/generator/src/parser.ts#L98-L105